lib/chef/chef_fs/file_system.rb

```diff
@@ -290,6 +292,8 @@ def copy_entries(src_entry, dest_entry, new_dest_parent, recurse_depth, options,
                       dest_entry.delete(true)
                       ui.output "Deleted extra entry #{dest_path} (purge is on)" if ui
                     rescue Chef::ChefFS::FileSystem::NotFoundError
+                      failure = {"src_path" => src_path, "reason" => "Entry #{dest_path} does not exist"}
+                      result["failed"].append(failure)
```

Comment:
@vinay-satish looks like there are still style errors like the above that still need to be corrected.
`rake style` to check and `rake style:auto_correct` to correct auto-correctable ones.